static/app/utils/discover/fieldRenderers.tsx

```diff
@@ -338,7 +348,30 @@ const SPECIAL_FIELDS: SpecialFields = {
             }}
             items={items}
           />
-        </Container>
+        </RightAlignedContainer>
+      );
+    },
+  },
+  minidump: {
+    sortField: null,
+    renderFunc: data => {
+      const attachments: Array<IssueAttachment & {url: string}> = data.attachments;
+
+      const minidump = attachments.find(
+        attachment => attachment.type === 'event.minidump'
+      );
+
+      return (
+        <RightAlignedContainer>
+          <Button size="xs" disabled={!minidump ? true : false}>
+            <IconDownload
+              onClick={() => window.open(minidump?.url)}
+              color="gray500"
```

Comment:
do we have that color in the theme? color as a variable would be easier to manage rather that a hardcoded one

Reply:
this isn't critical but should be addressed soon

Reply:
👍 Yup, got a couple of cleanup items for this code like this don't change the logic, was planning on getting to it once the tab is verfied throughly in prod

Reply:
Fixed